### PR TITLE
Calvin dev

### PR DESCRIPTION
**Pull request by calvincarter**
Here are the latest server-profile changes for PA Cluster and baseline

### CODE REVIEW

File: pa-cluster/hooks/50-before-post-start.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#
+
+# shellcheck source=pingcommon.lib.sh
+. "${HOOKS_DIR}/pingcommon.lib.sh"
+
+echo `set`
+
+echo_green 'Checking if OPERATIONAL_MODE is set...'
+echo "OPERATIONAL_MODE:"${OPERATIONAL_MODE}
+INITIAL_ADMIN_PASSWORD=${INITIAL_ADMIN_PASSWORD:=2FederateM0re}
```
Comment (savitha-ping) [inline pa-cluster/hooks/50-before-post-start.sh]: INITIAL_ADMIN_PASSWORD is set to 2FederateM0re in the docker image, so this assignment can be removed.

File: pa-cluster/hooks/50-before-post-start.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+if [[ ! -z "${OPERATIONAL_MODE}" && "${OPERATIONAL_MODE}" = "CLUSTERED_CONSOLE" ]]; then
+  echo "Shutting down the eth01 interface..."
+  ip link set eth0 down
```
Comment (savitha-ping) [inline pa-cluster/hooks/50-before-post-start.sh]: Why is this necessary?

File: pa-cluster/hooks/51-add-engine.sh [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+    # Download Engine Configuration
+    echo "EngineId:"${engineId}
+    echo "Retrieving the engine config"
+    make_api_request -X POST https://${pahost}:9000/pa-admin-api/v3/engines/${engineId}/config -o engine-config.zip
+
+    echo "Extracting config files to conf folder..."
+    unzip -o engine-config.zip -d ${OUT_DIR}/instance
+    ls -la ${OUT_DIR}/instance/conf
+    cat ${OUT_DIR}/instance/conf/bootstrap.properties
```
Comment (savitha-ping) [inline pa-cluster/hooks/51-add-engine.sh]: Does this file have secrets? If so, I wouldn't cat it.

File: pa-cluster/hooks/80-post-start.sh [hunk 4]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env sh
+#
+# Ping Identity DevOps - Docker Build Hooks
+#
+#- This script is used to import any configurations that are
+#- needed after PingAccess starts
+
+# shellcheck source=pingcommon.lib.sh
+. "${HOOKS_DIR}/pingcommon.lib.sh"
+INITIAL_ADMIN_PASSWORD=${INITIAL_ADMIN_PASSWORD:=2FederateM0re}
```
Comment (savitha-ping) [inline pa-cluster/hooks/80-post-start.sh]: This assignment should be removed. We want to keep this branch free of secrets. Besides INITIAL_ADMIN_PASSWORD is already set to 2FederateM0re in the PA image.

File: pa-cluster/hooks/50-before-post-start.sh (hunk 2 above)
Comment (savitha-ping) [reply pa-cluster/hooks/50-before-post-start.sh]: Since the console and engines are in two different StatefulSets in clustered mode, could this present a problem while adding engines and the admin is down? If so, are you handling it?

Edit: I see you wait in a loop for the admin to be ready, which is good.

File: pa-cluster/hooks/81-import-initial-configuration.sh [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  if ! test $? -eq 0 ; then
+    echo "Import Config: Server not started, waiting.."
+    sleep 3
+  else
+    echo "PA started, begin import"
+    break
+  fi
+done
+set -x
+INITIAL_ADMIN_PASSWORD=${INITIAL_ADMIN_PASSWORD:=2FederateM0re}
```
Comment (savitha-ping) [inline pa-cluster/hooks/81-import-initial-configuration.sh]: Remove assignment per previous comments

File: pa-cluster/hooks/51-add-engine.sh (hunk 3 above)
Comment (calvincarter-ping) [reply pa-cluster/hooks/51-add-engine.sh]: Good catch I removed. Was using for testing.
